server.py

```
@@ -1046,11 +1102,167 @@ def reply_to_post(
+            "success": True,
+            "action": action,
+            "subreddit": display_name,
+            "message": message,
+            "metadata": {
+                "timestamp": time.time(),
+                "subscribers": getattr(sub, "subscribers", None)
+            }
+        }
+
+    except Exception as e:
+        logger.error(f"Error {action}ing r/{clean_name}: {e}")
```

Ruff (0.14.8)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)